mantle/kola/tests/misc/network.go

```diff
@@ -310,74 +324,88 @@ var (
 
 	setupOvsScript = `#!/usr/bin/env bash
           set -ex
+
+          rm -rf /etc/NetworkManager/system-connections/*
```

Comment:
Did you mean to empty the `NMPATH` repo instead to match the other setup script?

Also, I probably don't have full context on this, but usually this wouldn't be something we'd be able to safely do on a customer's machine.